### PR TITLE
Arcticus config

### PR DESCRIPTION
**Pull request by jrood-nrel**
This also updates Spack. I think we probably want https://github.com/spack/spack/pull/27846 . I realize this will have an effect on the tests when we update, but I guess it should also help test the latest Spack.

### CODE REVIEW

File: scripts/install-exawind.sh [hunk 1]
```diff
@@ -34,15 +37,17 @@ else
   cmd "spack manager create-env -s exawind+hypre -d ${SPACK_MANAGER}/environments/exawind"
 fi
 
-cmd "cd ${SPACK_MANAGER}/environments/exawind"
-cmd "spack env activate -d ."
+printf "\nActivating Spack environment...\n"
+cmd "spack env activate -d ${SPACK_MANAGER}/environments/exawind"
 
 if [ "${SPACK_MANAGER_MACHINE}" == 'darwin' ]; then
```
Comment (psakievich) [inline scripts/install-exawind.sh:40]: We actually don't need to call this. If compilers are missing the environment will call this for us

File: configs/arcticus/env.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -l
+
+module use /soft/modulefiles
+module load intel/oneapi/release/2021.4.0
```
Comment (psakievich) [inline configs/arcticus/env.sh:4]: Did you have to build these or are they on the system?

**Review by psakievich — APPROVED** [review]
Looking good.

File: scripts/install-exawind.sh (hunk 1 above)
Comment (jrood-nrel) [reply scripts/install-exawind.sh:40]: Oh interesting. I think I will leave it explicit for now, but add a comment.

File: configs/arcticus/env.sh (hunk 2 above)
Comment (jrood-nrel) [reply configs/arcticus/env.sh:4]: These are on the machine.

**Pull request merged**